locales/en/messages.json

```diff
@@ -1788,6 +1791,12 @@
     "pidTuningFeedforwardTransitionHelp": {
         "message": "With this parameter, the Feedforward term can be reduced near the center of the sticks, which results in smoother end of flips and rolls.<br>The value represents a point of stick deflection: 0 - stick centered, 1 - full deflection. When the stick is above that point, Feedforward is kept constant at its configured value. When the stick is positioned below that point, Feedforward is reduced proportionally, reaching 0 at the stick center position.<br>Value of 1 gives maximum smoothing effect, while value of 0 keeps the Feedforward fixed at its configured value over the whole stick range."
     },
+    "pidTuningFeedforwardJitter": {
+        "message": "Jitter Reduction"
+    },
+    "pidTuningFeedforwardJitterHelp": {
+        "message": "With this parameter, the Feedforward term can be reduced near the center of the sticks, which results in smoother end of flips and rolls.<br>The value represents a point of stick deflection: 0 - stick centered, 1 - full deflection. When the stick is above that point, Feedforward is kept constant at its configured value. When the stick is positioned below that point, Feedforward is reduced proportionally, reaching 0 at the stick center position.<br>Value of 1 gives maximum smoothing effect, while value of 0 keeps the Feedforward fixed at its configured value over the whole stick range."
```

Comment:
This message applies for Transition; jitter reduction is very different and needs a different message.  Maybe:
"Jitter reduction reduces Feedforward when the sticks move slowly.  This allows smooth, jitter-free flight when making smooth slow arcs, yet provides full strong feedforward when the sticks move quickly.  A higher threshold value (10-12) is more useful for cinematic or HD freestyle purposes, and a slightly lower value (5) better for racing or higher speed RC links."

Reply:
Updated.